lib/views/login_register/forgot_password.dart

```diff
@@ -59,38 +45,19 @@ class ForgotPasswordScreen extends StatelessWidget {
                       borderRadius: BorderRadius.circular(12)),
                   child: const TextField(
                     decoration: InputDecoration(
-                        hintText: 'Email',
-                        hintStyle: TextStyle(color: Colors.grey),
-                        border: InputBorder.none,
-                        labelStyle: TextStyle(color: Colors.black)),
+                      hintText: 'Email',
+                      hintStyle: TextStyle(color: Colors.grey),
+                      border: InputBorder.none,
+                      labelStyle: TextStyle(color: Colors.black),
+                    ),
                     style: TextStyle(color: Colors.black),
                   ),
                 ),
                 Padding(
                   padding: const EdgeInsets.all(30),
-                  child: GestureDetector(
-                    onTap: () {
-                      Navigator.push(
-                        context,
-                        MaterialPageRoute(
-                          builder: (context) => const PasswordResetSentScreen(),
-                        ),
-                      );
-                    },
-                    child: Container(
-                      height: 50,
-                      margin: const EdgeInsets.symmetric(horizontal: 50),
-                      decoration: BoxDecoration(
-                        borderRadius: BorderRadius.circular(50),
-                        color: Colors.cyan,
-                      ),
-                      child: const Center(
-                        child: Text(
-                          'Reset Password',
-                          style: TextStyle(color: ThcColors.darkBlue),
-                        ),
-                      ),
-                    ),
+                  child: BigButton(
+                    onPressed: () => navigator.push(const PasswordResetSentScreen()),
+                    label: 'Reset Password',
```

Comment:
A button widget like `FilledButton` has a few advantages over a `GestureDetector`, including some subtle visual effects as you hover your mouse and click on it.

I'm a big fan of the big cyan button—I decided to make a widget for it so we don't have to repeatedly configure the size & shape.